<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-ABAS1HV8" ignored="0" name="Default test" tests="2" failures="0" timestamp="20 Feb 2019 17:35:37 GMT" time="0.16" errors="0">
  <testcase name="@BeforeTest initialize" time="0.031" classname="Academy.HomePage">
    <failure type="java.io.FileNotFoundException" message="C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\rahul\E2EProject\src\main\java\resources\data.properties (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at java.io.FileInputStream.<init>(Unknown Source)
at resources.base.initializeDriver(base.java:25)
at Academy.HomePage.initialize(HomePage.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest initialize -->
  <testcase name="@AfterTest teardown" time="0.0" classname="Academy.HomePage">
    <skipped/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase name="basePageNavigation" time="0.0" classname="Academy.HomePage">
    <skipped/>
  </testcase> <!-- basePageNavigation -->
  <testcase name="basePageNavigation" time="0.0" classname="Academy.HomePage">
    <skipped/>
  </testcase> <!-- basePageNavigation -->
</testsuite> <!-- Default test -->
